packages/fx-core/templates/plugins/resource/apiconnector/sample/headerComment.template

```diff
@@ -1,3 +1,5 @@
+// This file is auto generated by Teams Toolkit to provide you instructions and reference code to call your api.
```

Comment:
@timngmsft can you also help review the wording here?

Reply:
I would capitalize API.